.github/workflows/ci.yml

```diff
@@ -29,4 +29,4 @@ jobs:
           cd jpo-ode
           mvn install -DskipTests
           cd ../jpo-geojsonconverter
-          mvn -e -X clean org.jacoco:jacoco-maven-plugin:prepare-agent package sonar:sonar -Dsonar.projectKey=usdot-jpo-ode_jpo-geojsonconverter -Dsonar.projectName=jpo-geojsonconverter -Dsonar.organization=usdot-jpo-ode-1 -Dsonar.host.url=https://sonarcloud.io -Dsonar.branch.name=$GITHUB_REF_NAME
+          mvn -e -X clean org.jacoco:jacoco-maven-plugin:prepare-agent package sonar:sonar -Dsonar.projectKey=usdot-jpo-ode_jpo-geojsonconverter -Dsonar.projectName=jpo-geojsonconverter -Dsonar.organization=usdot-jpo-ode -Dsonar.host.url=https://sonarcloud.io -Dsonar.branch.name=$GITHUB_REF_NAME
```

Comment:
Why are we passing in the sonar organization in our maven build argument for some repos and for others we have it included in the pom.xml. Could you create a devops issue to make sure all ODE branches incorporate sonar scanner in a  similar way. Does not need to be addressed in this PR but should be addressed eventually.